chromadb/server/fastapi/__init__.py

```diff
@@ -341,7 +341,7 @@ def auth_and_get_tenant_and_database_for_request(
         if not self.authn_provider:
             return (tenant, database)
 
-        user_identity = self.authn_provider.authenticate_or_raise(headers)
+        user_identity = self.authn_provider.authenticate_or_raise(dict(headers))
```

Comment:
why this change?

Reply:
This is to fix a header check that was previously `self._token_transport_header.value not in headers`, which doesn't work with dict.